sentry_sdk/tracing.py

```diff
@@ -606,6 +607,12 @@ def finish(self, hub=None):
             "spans": finished_spans,
         }
 
+        if hub.client.options["enable_profiling"]:
```

Comment:
remove the `except` below and just add a condition here
```suggestion
        if hub.client.options["enable_profiling"] and self._profile:
```

Reply:
I changed this to `and self._profile is not None` because I thought that was more readable; is that OK?

Reply:
yes that's fine